pkg/services/ngalert/state/manager.go

```diff
@@ -37,19 +37,22 @@ type Manager struct {
 	images        ImageCapturer
 	historian     Historian
 	externalURL   *url.URL
+
+	doNotKeepNormalState bool
 }
 
-func NewManager(metrics *metrics.State, externalURL *url.URL, instanceStore InstanceStore, images ImageCapturer, clock clock.Clock, historian Historian) *Manager {
+func NewManager(metrics *metrics.State, externalURL *url.URL, instanceStore InstanceStore, images ImageCapturer, clock clock.Clock, historian Historian, doNotKeepNormalState bool) *Manager {
```

Comment:
`doNotKeepNormalState` == `skipNormalState`? This is another reason why i'd prefer just using the toggle directly, it's less ambiguous and less intrusive to the existing code.

Reply:
renamed to DoNotSaveNormalState, which I think more close to what it does